packages/angular/ssr/src/routes/ng-routes.ts

```diff
@@ -478,7 +478,12 @@ export async function getRoutesFromAngularRouterConfig(
         }
       }
     } else {
-      routesResults.push({ route: '', renderMode: RenderMode.Prerender });
+      const renderMode = serverConfigRouteTree?.match('')?.renderMode ?? RenderMode.Prerender;
```

Comment:
@dgp1130, I'd prefer if in this case it was a hard error both the old and new API. But this would require us to generate an additional component and add route when doing `ng-new`. 

Is this something that we'll be willing to do?

Reply:
This is the case where `export const SERVER_ROUTES = [];`? You think that should be a hard error?

I can see where you're coming from. This has bugged me in the past because we do the same thing for client-side routes and it means the `<router-outlet>` is a somewhat confusing no-op in `ng new` and it's not clear how to add a route, which leads to devs using `component` instead of `loadComponent` and deoptimizing their apps.

Could we generate `{ path: '/**', renderMode: RenderMode.Prerender }` by default in SSR? Would that avoid the need for a defined route? Or do we need to add a route in the app router to get this to work?